docs/reference/team-processes/release-process.md

```diff
@@ -2,146 +2,177 @@
+        <img src={regularReleaseDiagram} />
+
+    An example of what your commands may look like:
 
-1. Run [release.sh][release.sh] from the root of the repository.  Make sure there are no errors in the output.
+        $ git checkout main
+        $ git pull
+        $ ./release.sh
+        # check `git log`, `git diff`, then follow instructions
+    </details>
 
 1. Do some manual checks in the new train branch to make sure the generated tags are sane:
 
-    1. Do the changelogs match expectations from `git log`?
-    1. Have all the version strings been updated?
-    1. Does the diff from `origin/main` (or `origin/train-xxx` if it’s a point release) look correct?
+    1. Does `git log` look as expected, showing all commits between the previous tag, and the new tag?
+    1. When running a `git diff` from `origin/main`, does the diff look correct, including:
+        - Do the changelogs look as expected with a version bump and commit message summaries?
+        - Have all the version strings in `package.json` been updated?
+        - Do the files changed look as expected for the major release?
 
-1. The release script will print some commands to run to push the train branch to the server.  **It's best to copy and paste these so you don't mix them up.**
+1. The release script will print some commands to run to push the train branch and the tag to the repo. You might consider doing all other commands in a new terminal tab to not lose the print out. **It's best to copy and paste the push commands so you don't mix them up.** The deploy to staging will begin once `deploy_and_tag` finishes in CI which runs after all tests have passed.
 
-1. The release script will also print a URL which you can use to open a PR to merge the train branch back into the main branch
+1. The release script will also print a URL which you can use to open a PR to merge the train branch back into the main branch.
 
-1. Finally, the release script will print out a bug template.  Copy that template and open a deployment bug in bugzilla under `Cloud Services :: Operations: Deployment Requests` ([example][example-deployment-bug]) even if the component says it's deprecated. Remember to include:
+1. The release script will print out a bug template.  Copy that template and open a deployment bug in bugzilla under `Cloud Services :: Operations: Deployment Requests` ([example][example-deployment-bug]) even if the component says it's deprecated. Remember to include:
 
-    1. Notes from the deploy doc, particularly any server side changes that need to happen as part of this deployment.
-    1. Links to the needs:qa label on GitHub.
+    1. Any notes that were added during the sprint from the [deploy doc][deployment-doc] for the train, particularly any server side changes that need to happen as part of this deployment.
+    1. Links to the `needs:qa` label on GitHub.
```

Comment:
SubPlat uses `qa+` and `qa-` labels (the deploy bug template has already been updated).

Reply:
This is a good catch.  We should all be using `qa+` and should update that link

Reply:
@biancadanforth do you mark _every_ issue with `qa-` if no testing is needed?

Reply:
When I remember, hehe. Otherwise sometimes QA pings the dev in the ticket and asks about it, so it's to avoid confusion.

IIUC these `qa+`/`qa-` labels are for one-offs rather than feature work, where we have a detailed test plan and test cases.